cmd/prysmctl/testnet/generate_genesis.go

```diff
@@ -86,30 +97,25 @@ type depositDataJSON struct {
 }
 
 func cliActionGenerateGenesisState(cliCtx *cli.Context) error {
-	if !cliCtx.IsSet(genesisTimeFlag.Name) {
-		log.Infof("No %s specified, defaulting to now()", genesisTimeFlag.Name)
-	}
-	hasOutputFlag := !cliCtx.IsSet(outputJsonFlag.Name) &&
-		!cliCtx.IsSet(outputYamlFlag.Name) &&
-		!cliCtx.IsSet(outputSSZFlag.Name)
-	if !hasOutputFlag {
-		return fmt.Errorf(
-			"no %s, %s, or %s flag specified. At least one is required",
-			outputJsonFlag.Name,
-			outputYamlFlag.Name,
-			outputSSZFlag.Name,
+	if generateGenesisStateFlags.GenesisTime == 0 {
+		log.Info("No genesis time specified, defaulting to now()")
+	}
+	outputJson := generateGenesisStateFlags.OutputJSON
+	outputYaml := generateGenesisStateFlags.OutputYaml
+	outputSSZ := generateGenesisStateFlags.OutputSSZ
+	noOutputFlag := outputSSZ == "" && outputJson == "" && outputYaml == ""
+	if noOutputFlag {
+		return errors.New(
+			"no --output-ssz, --output-json, or --output-yaml flag specified. At least one is required",
```

Comment:
One last comment: can you parameterize flag names for easier maintenance? In case names change in the future?